en/flight_modes/follow_me.md

```diff
@@ -41,45 +49,63 @@ The following flight precautions should be observed:
 - Give your vehicle sufficient room to stop, especially when it is moving fast.
-  Many Android devices do not update their position very frequently, and autopilot estimations of the speed and direction can be inaccurate.
-- Be ready to take manual RC control if something goes wrong when using follow me mode for the first time.
-  The accuracy of positioning is dependent on the quality of the GPS used by target system.
-  If the GPS is not accurate, this will be reflected in follow me.
+- Be ready to switch back to Position mode, etc, if something goes wrong when using follow me mode for the first time.
+  If the target system (e.g. Android Phone)'s GPS is not accurate, this can affect the behavior.
 
-## Using Follow Me with QGroundControl
+## How to use Follow Me using QGroundControl
 
 *Follow Me* mode is supported by *QGroundControl* on ground station hardware that has a GPS module.
 The recommended configuration is a USB OTG-capable Android device with two telemetry radios.
 
 To setup *Follow Me* mode:
-- Connect a telemetry radio to your Android device and another to the vehicle (this allows positioning information to be relayed between the two radios).
+- Connect a telemetry radio to your ground station device and another to the vehicle (this allows positioning information to be relayed between the two radios).
 - Disable sleep-mode on your Android device:
   - This setting can usually be found under: **Settings \> Display**.
   - It is important that you set your Android device to not go to sleep as this could cause the GPS signal to cease being emitted at regular intervals.
 - Takeoff to a height of at least 2-3 metres (recommended even though auto-takeoff is supported).
   - Set the vehicle on the ground, press the safety switch and step back at least 10 meters.
   - Arm the vehicle and takeoff.
 - Switch into follow me mode.
-  - The copter will ascend to the specified [minimum height](#NAV_MIN_FT_HT) and then pause for a moment to assess the radio link.
-    If the link update rate is OK the multicopter will then yaw to face the target.
+  - The copter will first ascend to minimum safety altitude of 1 meters above the ground or home, depending on the presence of a distance sensor.
+  - It will ascend until it is 3 meters within the [follow height](#FLW_TGT_HT) to avoid potential collisions before moving horizontally.
+  - Copter will always adjust it's heading to face the target
 
-At this point you should be able to start moving and the copter should follow your movements.
+At this point you can start moving, and the drone will be following you.
 
 The mode has been tested on the following Android devices:
-- Nexus 5
+- Galaxy S10
 - Nexus 7 Tablet
 
+## Using MAVSDK to use Follow Me
+MAVSDK supports [Follow Me](https://mavsdk.mavlink.io/main/en/cpp/guide/follow_me.html). However it currently has a bug where a same message is getting sent twice occasionally, which can mess with the target position and velocity estimator.

```

Comment:
Provide a bug link here

Reply:
I have reported just now : https://github.com/mavlink/MAVSDK/issues/1756
Hoping it will get solved before merging this doc :pray:

Reply:
I reworded this and add the bug number. It is better to have the bug issue there and forget to remove it than merge this without adding it.